sha2/tests/mod.rs

```diff
@@ -1,14 +1,61 @@
-use digest::dev::{feed_rand_16mib, fixed_reset_test};
-use digest::new_test;
+use digest::{
+    crypto_common::SerializableState,
+    dev::{feed_rand_16mib, fixed_reset_test},
+    new_test,
+};
 use hex_literal::hex;
 use sha2::{Digest, Sha224, Sha256, Sha384, Sha512, Sha512_224, Sha512_256};
 
+macro_rules! test_serializable_state {
+    ($name:ident, $hasher:ty) => {
```

Comment:
You should add tests which check stability of the serialized state, i.e. that every version of this crate results in the same serialized state for a given hasher state.

Reply:
Test macro moved to `traits` and added `expected_serialized_state`. https://github.com/RustCrypto/traits/pull/1078/commits/793f9f268d220be0420e2c6b285a367ade048f88.